core/src/main/resources/hudson/model/Messages_pl.properties

```diff
@@ -37,7 +38,7 @@ Run.Summary.BrokenSinceThisBuild=nieudany od tego zadania
 Run.Summary.BrokenSince=nieudany od zadania {0}
 Run.Summary.Unknown=?
 
-UpdateCenter.DownloadButNotActivated=Pobrany pomy\u015Blnie. Zostanie w\u0142\u0105czony podczas kolejnego restartu Jenkinsa
+UpdateCenter.DownloadButNotActivated=Pobrana pomy\u015Blnie. Zostanie w\u0142\u0105czony podczas kolejnego restartu Jenkinsa
```

Comment:
Since 'pobrany' was changed to 'pobrana', the following 'włączony' should be updated to 'włączona' as well.